distribution/lib/Standard/Table/0.0.0-dev/src/Excel/Excel_Workbook.enso

```diff
@@ -289,12 +297,50 @@ type Excel_Workbook
 
     ## PRIVATE
     with_java_workbook : (Workbook -> Any) -> Any ! Illegal_State
-    with_java_workbook self f = Illegal_State.handle_java_exception <|
+    with_java_workbook self f = Illegal_State.handle_java_exception <| handle_invalid_location <| Empty_Sheet.handle_java_exception <|
         case self.excel_connection_resource_ref.get of
             Nothing ->
                 Error.throw (Illegal_State.Error "The workbook is being used after it was closed.")
             resource -> resource.with connection->
                 connection.withWorkbook f
 
+    ## GROUP Standard.Base.Input
+       ICON data_input
+       Reads one or more sheets into a Table.
+
+       Arguments:
+       - sheet_names: the names of the sheets to read. Defaults to all sheets.
+         An invalid `sheet_name` will be reported as a problem.
```

Comment:
```suggestion
       - sheet_names: the names of sheets, ranges or addresses to read. Defaults to all sheets.
         An invalid `sheet_name` will be reported as a problem.
```

(or something like that)

As I mentioned on our call, I think it's good to make it clear from the docs that this argument also accepts addresses. The name `sheet_names` is OK as it's how it will be used most of the time, but given that name, without a note in the doc, I would **_never_** guess that this can accept also an address.

----

It may also be beneficial to add an example where a custom range is read as well, e.g. `workbook.read_many ["Sheet1", "Sheet2!A2:C100"]` to illustrate clearly how to use this